sample-windows.ini

```diff
@@ -14,9 +14,11 @@ portNumber = 8000
 remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
 ; These options are unlikely to need editing by users
-maxCacheEntries = 2000000
-mruSymbolStateFile = %(TEMP)s\snappy-mru-symbols.json
-maxMRUSymbolsPersist = 10
+maxCacheEntries = 400
```

Comment:
The Windows symbol server is for profiling local builds where users might rebuild their libraries and we'll be symbolicating based on outdated symbols.

Can you add a switch so that symbols read from local disk do not get cached in either cache. Do this in a different pull request.